tools/cpp_build/libtorch/CMakeLists.txt

```diff
@@ -141,51 +139,34 @@ TARGET_LINK_LIBRARIES(torch
   ${ATEN_LIBRARY}
 )
 
-TARGET_INCLUDE_DIRECTORIES(torch
-  PUBLIC
+set(COMMON_INCLUDES
   "${ATEN_INCLUDE_DIR}"
   "${ATEN_INCLUDE_DIR}/TH"
+  "${ATEN_BUILD_INCLUDE_DIR}"
+  "${ATEN_BUILD_PATH}/src/TH"
   "${CMAKE_CURRENT_SOURCE_DIR}"
+  "${CUDA_INCLUDE_DIRS}")
+
+TARGET_INCLUDE_DIRECTORIES(torch
+  PUBLIC
+  "${COMMON_INCLUDES}"
   "${TORCH_SRC_DIR}/../"
-  "${TORCH_SRC_DIR}/../third_party/nanopb"
-  "${CUDA_INCLUDE_DIRS}"
-  )
+  "${TORCH_SRC_DIR}/csrc/api/include"
```

Comment:
I can't find this directory in the source tree, is it a new one?

Reply:
yeah sorry that's a vestige from the C++ API in a separate PR, I will remove it -- https://github.com/pytorch/pytorch/pull/6345/files